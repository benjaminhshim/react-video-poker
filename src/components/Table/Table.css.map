{"version":3,"sourceRoot":"","sources":["Table.scss"],"names":[],"mappings":"AAAA;AAAA;EAEI;EACA;EACA","file":"Table.css"}