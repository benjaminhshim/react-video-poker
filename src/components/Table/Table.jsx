import React, { Component } from 'react';

import {Button} from 'mdbreact';

export default class Table extends Component {
  render() {
    return (
      <div className="btn-container">
        {this.props.game ? 
        <Button 
          id="go-btn"
          color="danger"
          onClick={this.props.newHand}>
          GO
        </Button>
      :
        <Button 
          id="deal-btn"
          color="yellow"
          onClick={this.props.dealCards}>
          DEAL
        </Button>
      }
        
      </div>
    )
  }
}
