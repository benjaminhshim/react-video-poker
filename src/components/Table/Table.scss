#go-btn, 
#deal-btn {
    font-size: 1.2rem;
    letter-spacing: 0.05rem;
    padding: 0.7rem 2.5rem;
}