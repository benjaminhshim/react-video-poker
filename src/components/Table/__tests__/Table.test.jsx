import React from 'react';
import { shallow, mount, render, configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import Table from '../Table.jsx';

configure({ adapter: new Adapter() });

describe('Table', () => {
 it('should be defined', () => {
   expect(Table).toBeDefined();
 });


});