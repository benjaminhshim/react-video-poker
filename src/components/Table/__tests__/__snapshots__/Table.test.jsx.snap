// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Table should render correctly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Table />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <t
        color="yellow"
        tag="button"
      >
        DEAL
      </t>,
      "className": "btn-container",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": "DEAL",
        "color": "yellow",
        "onClick": undefined,
        "tag": "button",
      },
      "ref": null,
      "rendered": "DEAL",
      "type": [Function],
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <t
          color="yellow"
          tag="button"
        >
          DEAL
        </t>,
        "className": "btn-container",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": "DEAL",
          "color": "yellow",
          "onClick": undefined,
          "tag": "button",
        },
        "ref": null,
        "rendered": "DEAL",
        "type": [Function],
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "attachTo": undefined,
    "hydrateIn": undefined,
  },
}
`;
